### ISSUE TITLE
Runtime error when loading frozen graph from Tensorflow

### ISSUE THREAD

**Issue by alecda573** [opencv/opencv#17151]
##### Detailed description

I am trying to serve the following gitrepo in opencv: https://github.com/una-dinosauria/3d-pose-baseline 
and the checkpoint data can be found at the following link:
https://drive.google.com/file/d/0BxWzojlLp259MF9qSFpiVjl0cU0/view

I have already constructed a frozen graph which I can serve in python and was generated using the following script:

    meta_path = 'checkpoint-4874200.meta' # Your .meta file
    output_node_names = ['linear_model/add_1']    # Output nodes
    export_dir=os.path.join('export_dir')
    graph=tf.Graph()

    with tf.Session(graph=graph) as sess:
    # Restore the graph
        loader=tf.train.import_meta_graph(meta_path)
        loader.restore(sess,'checkpoint-4874200')
    
        builder=tf.saved_model.builder.SavedModelBuilder(export_dir)
        builder.add_meta_graph_and_variables(sess,
                                         [tf.saved_model.SERVING],
                                         strip_default_attrs=True)
    

    # Freeze the graph
    frozen_graph_def = tf.graph_util.convert_variables_to_constants(
        sess,
        sess.graph_def,
        output_node_names)

    # Save the frozen graph
    with open('C:\\Users\\FrozenGraph.pb', 'wb') as f:
      f.write(frozen_graph_def.SerializeToString())

and the frozen graph can be found at:
https://github.com/alecda573/frozen_graph/blob/master/FrozenGraph.pb

When I try to run in opencv the following I get this runtime error:

OpenCV(4.3.0) Error: Unspecified error (More than one input is Const op) in cv::dnn::dnn4_v20200310::`anonymous-namespace'::TFImporter::getConstBlob, file C:\build\master_winpack-build-win64-vc15\opencv\modules\dnn\src\tensorflow\tf_importer.cpp, line 570

##### Steps to reproduce

To reproduce problem you just need to download the frozen graph from the above link or create yourself from the checkpoint data and then call the following in opencv with the below headers: 

     #include <iostream>
     #include <vector>
     #include <cmath>
     #include <opencv2/core/core.hpp>
     #include <opencv2/highgui/highgui.hpp>
     #include <opencv2/imgproc.hpp>
     #include "opencv2/dnn.hpp"

	string pbFilePath = "C:/Users/FrozenGraph.pb";
	//Create 3d-pose-baseline model
	cv::dnn::Net inputNet;
	inputNet = cv::dnn::readNetFromTensorflow(pbFilePath);

Would love to know if anyone has any thoughts on how to address this error.

**Comment by alecda573** [comment]
![png(1)](https://user-images.githubusercontent.com/60713416/80287328-93ebc380-86f6-11ea-9063-935fd5d807f9.png)

**Comment by alecda573** [comment]
Is the error regarding the isTraining and dropout flags?

**Comment by alecda573** [comment]
Ran the following optimize for inference script and will attach the optimized graph, and I am still catching the same error which makes me think it is the isTraining and dropout flag that are causing the issues:

    optimized_graph_def=optimize_for_inference_lib.optimize_for_inference(
        frozen_graph_def,
        ['inputs/enc_in'],
        ['linear_model/add_1'],
        tf.float32.as_datatype_enum)
    g=tf.gfile.FastGFile('optimized_inference_graph.pb','wb')
    g.write(optimized_graph_def.SerializeToString())

![png(2)](https://user-images.githubusercontent.com/60713416/80289930-9acf0200-8707-11ea-9fc9-3f5727dd696f.png)

**Comment by dkurt** [comment]
@alecda573, Hi, sorry for late response.

Is there a way to save a checkpoint / graph in testing mode? By `set_learning_phase(0)` if Keras is used, in example.

**Comment by alecda573** [comment]
@dkurt it is already set in testing mode. What were you thinking?

**Comment by alecda573** [comment]
@dkurt any ideas to what is going on?
